src/pages/Locations/LocationInfo/LocationInfo.tsx

```diff
@@ -0,0 +1,54 @@
+import type { FC } from 'react';
+import { IonList, IonItem, IonIcon, IonLabel, IonText } from '@ionic/react';
+import '../LocationItem.css';
+import { callOutline, globeOutline, locationOutline, mailOutline } from 'ionicons/icons';
+import type { Location } from '../../../types';
+
+interface LocationProp {
+  location: Location | undefined;
+}
+
+const LocationInfo: FC<LocationProp> = ({ location }: LocationProp) => {
```

Comment:
suggestion: you can create the array structure of icon and label iterate over that to render the list.